arcade/future/video/video_player.py

```diff
@@ -41,7 +41,7 @@ def draw(self, left: int = 0, bottom: int = 0, size: tuple[int, int] | None = No
 
         Args:
             size: Pass None as one of the elements if you want to use the
-            dimension(width, height) attribute.
+             dimension(width, height) attribute.
```

Comment:
This one doesn't make sense :)

This is in the future module so it's kind of incomplete. The docstring is also terrible. `left` `right` is the position from the lower left corner an `size` is the size of the rectangle for the video contents.